stories/IssueSummary.vue

```diff
@@ -1,9 +1,7 @@
 <template>
 	<div class="issue-summary">
 					<div class="issue-summary__wrapper">
-						<div class="issue-summary__volume">
-							<h2>Vol {{ volume }}, No {{ number }}</h2>
-						</div>
+						<h2 class="issue-summary__volume">{{ volume }}, No {{ number }}</h2>
```

Comment:
Looks like the `Vol" text got dropped here. So it will say `1, No 1` instead of `Vol 1, No 1`.

Reply:
(Bungee-jumping in to say: shouldn't this be localized?)

Reply:
None of this is actually production code. This is a sandbox with test date and formulations for working out accessible markup/css. It will become translate-able when we work it into a theme.

So, point taken but we'll solve that problem down the line.

Reply:
Ah, of course, it's in `stores`. Thanks!